src/mono/sample/wasm/console-node-ts/main.ts

```diff
@@ -9,3 +9,6 @@ const { runMainAndExit } = await createDotnetRuntime(() => ({
 const app_args = process.argv.slice(2);
 const dllName = "Wasm.Console.Node.TS.Sample.dll";
 await runMainAndExit(dllName, app_args);
+
```

Comment:
needs more changes

Reply:
Yea, we were missing some types in export https://github.com/dotnet/runtime/pull/74166.